kernelcare/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,28 @@
+name: kernelcare
+files:
+- name: kernelcare.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - name: key
+      required: false
+      description: |
+        Kernelcare license key
+      value:
+        type: string
+    - name: login
+      required: false
+      description: |
+        Kernelcare login
```

Comment:
is this a username/password combo?

Reply:
for auth to our billing API we have 2 options:

1. Just use 'key'
2. Use pair 'login' + 'api_token'

let me know please if I need to improve spec.yaml for this